src/group_theory/group_action/option.lean

```diff
@@ -0,0 +1,55 @@
+/-
+Copyright (c) 2022 Yaël Dillies. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Yaël Dillies
+-/
+import group_theory.group_action.defs
+
+/-!
+# Option instances for additive and multiplicative actions
+
+This file defines instances for additive and multiplicative actions on `option` type. Scalar
+multiplication is defined by `a • some b = some (a • b)` and `a • none = none`.
+
+## See also
+
+* `group_theory.group_action.pi`
+* `group_theory.group_action.prod`
+* `group_theory.group_action.sigma`
+* `group_theory.group_action.sum`
+-/
+
+variables {M N P α β γ : Type*}
```

Comment:
```suggestion
variables {α β γ : Type*}
```
as far as I can tell, these aren't used

Reply:
Although in the other file, you use `M N P` for the action, and `α β γ` for the thing being acted on